kibana/templates/deployment.yaml

```diff
@@ -52,10 +52,39 @@ spec:
     {{- with .Values.nodeSelector }}
       nodeSelector:
 {{ toYaml . | indent 8 }}
-    {{- end }}
-    {{- with .Values.affinity }}
+      {{- end }}
+      {{- if or (eq .Values.antiAffinity "hard") (eq .Values.antiAffinity "soft") .Values.nodeAffinity }}
+      {{- if .Values.priorityClassName }}
```

Comment:
This is already defined at https://github.com/elastic/helm-charts/blob/741de11d2226bc07957cbdcc11660169650e7099/kibana/templates/deployment.yaml#L33-L35. Did you mean to duplicate this?